certify-service/src/main/resources/svg-templates.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file needs to have a static UUID by some means so that the exact ID can be hardcoded in this JSON here.

Reply:
In the json, multiple templates can be added with a unique name as the key so that all of them can be added in the db.